src/Command/ImportCommand.php

```diff
@@ -44,33 +50,21 @@ protected function configure()
 
         parent::configure(); // TODO: Change the autogenerated stub
     }
+
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
 
         $orgName = $input->getArgument('username');
         $provider = $input->getArgument('provider');
-        
-        if($this->checkProviderExistance($provider) && !$this->repoRepository->findOrgWithProvider($provider, $orgName)) {
-            switch(strtolower($provider)) {
-                case 'github':
-                    if(!$this->githubService->importToDb($orgName)) {
-                        return COMMAND::FAILURE;
-                    }
-                    break;
-                case 'bitbucket':
-                    if(!$this->bitbucketService->importToDb($orgName)) {
-                        return COMMAND::FAILURE;
-                    }
-                    break;
-            }
-
-            $output->writeln('Successfully imported ' . $orgName . ' into the ' . $provider . ' repository.');
-            return Command::SUCCESS;
-        }
-            $output->writeln($provider . ' is not implemented yet, or ' . $orgName . ' is already in DB with this provider');
-            return Command::FAILURE;
-        
-    }
-    
 
+
+        //$this->githubService->importToDb($orgName);
+
+        //$this->bitbucketService->importToDb($orgName);
+
+        $importCommand = $this->importCommandCreator->create($input->getArgument('username'), $input->getArgument('provider'));
```

Comment:
we can use try / catch construct 
```suggestion
        try {
            $importCommand = $this->importCommandCreator->create($input->getArgument('username'), 
            $input->getArgument('provider'));
        } catch (ProviderNotFound $exceptioin) {
            //display some message to the user that we dont have this provider
            return Command::FAILURE; 
        }
```